pages/signup.php

```diff
@@ -128,10 +128,36 @@
       <input type="text" name="email" placeholder="Email" class="requiredField">
       <input type="number" name="hosp_id" placeholder="Hospital ID" class="requiredField">
       <input type="password" name="pswd" placeholder="Password" id="pswd" class="requiredField">
-      <input type="password" name="re_pswd" placeholder="Re-enter Password" class="requiredField">
+      <input type="password" name="re_pswd" placeholder="Re-enter Password" id="pswd2" class="requiredField">
+      <div style="width:12vw;color:#fff;">
+                <input type="checkbox" id="box" onclick="box1()" style="width:auto;">
+                <span id="notice">show password</span>
+            </div>
       <button type="submit" name="submit" value="submit" id="submitBtn">Sign Up</button>
     </div>
       	
   </form>
   
-<?php include("../include/footer.inc.php") ?>
\ No newline at end of file
+<?php include("../include/footer.inc.php") ?>
+
+<script>
+function box1()
```

Comment:
This function seems to be a duplicate of the previous function (although there is a difference in the number of elements).

You can try using `regex` in `getElementById` for find the element with id `pswd` and `pswd2` (like `pswd | pswd2`). With this, u can take care of both the `login` and `signup` page show passwords in the same js function instead of two separate functions.